RAP/rap-statistics.qmd

```diff
@@ -785,23 +878,18 @@ For those who are interested in starting writing their own QA scripts, it's wort
 The [janitor](https://garthtarr.github.io/meatR/janitor.html){target="_blank" rel="noopener noreferrer"} package in R also has some particularly useful functions, such as `clean_names()` to automatically clean up your variable names, `remove_empty()` to remove any completely empty rows and columns, and `get_dupes()` which retrieves any duplicate rows in your data - this last one is particularly powerful as you can feed it specific columns and see if there's any duplicate instances of values across those columns.
```

Comment:
The link above on coffee and coding by Peter Curtis seems to be broken

Reply:
I couldn't find a link to the video anymore but have updated the link to the folder containing the resources